R/cr_cn.r

```diff
@@ -37,6 +37,14 @@
 #'   `http://data.medra.org`. DOI agency is checked first (see
 #'   [cr_agency()]).
 #'
+#' @section caching:
+#' 
+#' Caching is done using the package \pkg{webmiddens}. To cache requests
```

Comment:
Explain a bit more what caching means? "The first time you get results for a given DOI, the result is stored on disk (this is the cache), so that next time the same query is made, and until expiry of the cache, the result is read from disk without any web API call" or so? With a link to webmiddens docs if there's some general explanation there?